CONTRIBUTING-fil.md

```diff
@@ -0,0 +1,224 @@
+*Read this in other languages: [Deutsch](CONTRIBUTING-de.md), [Français](CONTRIBUTING-fr.md), **Filipino**, [Español](CONTRIBUTING-es.md), [Indonesia](CONTRIBUTING-id.md),[简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [Português (BR)](CONTRIBUTING-pt_BR.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
+
+## Kasunduan sa Lisensya ng Contributor
+Sa pamamagitan ng pag-aambag sumasang-ayon ka sa [LICENSE](https://github.com/EbookFoundation/free-programming-books/blob/master/LICENSE) ng repositoryong ito.
+
+## Kodigo ng Pag-uugali ng Contributor
+Sa pamamagitan ng pag-aambag sumasang-ayon kang igalang ang [Code of Conduct](https://github.com/EbookFoundation/free-programming-books/blob/master/CODE_OF_CONDUCT.md) ng repositoryong ito.
```

Comment:
Linking with related is safe since https://github.com/EbookFoundation/free-programming-books/pull/6524#issuecomment-955200952 is submitted too.

```suggestion
Sa pamamagitan ng pag-aambag sumasang-ayon kang igalang ang [Code of Conduct](https://github.com/EbookFoundation/free-programming-books/blob/master/CODE_OF_CONDUCT-fil.md) ng repositoryong ito.
```

Reply:
@davorpa we're siblings, we're trying to participate to this event, we share a single machine and we doing our best to contribute. Hope our entries count sir. thank you.

Reply:
I understand :+1: but it was a bit suspicious all family have the same cloned projects

Reply:
Not applied yet in https://github.com/EbookFoundation/free-programming-books/pull/6504/commits/915c4969ddfe89026fb069ebb4d050125a1a7926